pkg/gateway/db/pq/conn.go

```diff
@@ -0,0 +1,30 @@
+
+package pq
+
+import (
+	"context"
+
+	sq "github.com/lann/squirrel"
+	"github.com/jmoiron/sqlx"
+	"github.com/lib/pq"
+)
+
+var psql = sq.StatementBuilder.PlaceholderFormat(sq.Dollar)
+
+type conn struct {
+	DB                     *sqlx.DB
+	context                context.Context
+}
```

Comment:
I think we should change the organization to match with "database/sql" package.

For long time, I think conn has a DB is confusing with the `database/sql` package `DB` return `Conn`.